plugins/exporter.koplugin/clip.lua

```diff
@@ -228,6 +238,14 @@ end
 
 function MyClipping:parseHighlight(highlights, bookmarks, book)
     --DEBUG("book", book.file)
+
+    -- create a translated pattern that matches bookmark auto-text
+    -- see ReaderBookmark:getBookmarkAutoText and ReaderBookmark:getBookmarkPageString
```

Comment:
```suggestion
    -- create a translated pattern that matches bookmark auto-text
    -- see ReaderBookmark:getBookmarkAutoText and ReaderBookmark:getBookmarkPageString
    --- @todo GET RID OF ALL THIS JUNK
```

(possibly phrased slightly more politely in not all-caps)